### PR TITLE
Feature/terra compose

### PR DESCRIPTION
**Pull request by pixeleet**
Okay, I might have went a bit nuts on this one. 
Wanted to see if I could replace docker-compose with terraform so you wouldn't have to have docker-compose. 

Also I thought it might be nice that we can describe the docker parts in terraform too.

### CODE REVIEW

**Comment by ppvg** [comment]
Can `variables.tf` be `.gitignore`d?

**Comment by pixeleet** [comment]
It can be as long as we instruct the user to create it and/or provide a `variables.example.tf` like with consul config json.

File: docker-compose.yml [hunk 1]
```diff
@@ -19,3 +41,8 @@ vault:
     - consul:consul
   volumes:
     - vault/assets/conf.d:/vault/conf.d
+  environment:
+    - "SERVICE_NAME=vault"
+    - "SERVICE_TAGS=vault"
+    - "VAULT_ADDR=http://0.0.0.0:8200"
+    - "VAULT_TOKEN=lofasz"
```
Comment (ppvg) [inline docker-compose.yml]: Really?? :confused:

Comment (pixeleet) [inline docker-compose.yml]: docker-compose.yml needs some love it's a bit outdated. I'll throw a quick
fix in for that.

2015-10-27 10:58 GMT+01:00 Peter-Paul notifications@github.com:

Comment (pixeleet) [inline docker-compose.yml]: @ppvg: Updated docker-compose.yml.

2015-10-27 12:40 GMT+01:00 David Lonyai david.lonyai@gmail.com:

**Comment by pixeleet** [comment]
@ppvg Yo ppvg. Introduced a few nice changes.
- consul and nginx config files now can be on any path (by default the examples are copied into where docker expects them to be)
- removed linking where possible

**Comment by pixeleet** [comment]
@ppvg this branch is closed use [release/0.1](https://github.com/Sweebr/terra-con-vault/tree/release/0.1) branch and it's corresponding 
[PR](https://github.com/Sweebr/terra-con-vault/pull/2].

**Pull request closed without merging**